client/src/components/Instructions/index.js

```diff
@@ -1,103 +1,102 @@
 import React, { Component } from "react";
 import { Button } from "rimble-ui";
-import styles from './Instructions.module.scss';
+import styles from "./Instructions.module.scss";
 
 export default class Instructions extends Component {
-
   renderCounterSetup() {
     const addressDefault = this.getDefaultAddress();
     return (
       <div className={styles.instructions}>
         <h2> Build your first app with ZepKit </h2>
         <div className={styles.step}>
           <div className={styles.instruction}>
-            Add the Counter contract to your project.
+            Initialize your ZeppelinOS project
           </div>
           <div className={styles.code}>
-            <code>
-              zos add Counter
-            </code>
+            <code>zos init {"<<your project name>>"}</code>
           </div>
         </div>
         <div className={styles.step}>
           <div className={styles.instruction}>
-            Connect with your local blockchain by opening a session (Note that we automatically prefilled your ganache account).
+            Add the Counter contract to your project.
```

Comment:
I think since now we have interactive commands we should skip Add/Push commands all together because create command will do it all. WDYT?

Reply:
I think we can skip the Push commands, but we still need the Add commands, don't we?

Reply:
No we don't. Just `zos init project` and then `zos create Counter` 😎

Reply:
good!